evm/Cargo.toml

```diff
@@ -44,6 +44,8 @@ proptest = "1.0.0"
 # Display
 yansi = "0.5.1"
 url = "2.2.2"
+tui = { version = "0.16.0", default-features = false, features = ["crossterm"] }
```

Comment:
this is required for formatting?

any way we can move formatting to the UI crate entirely?

Reply:
i guess - i thought about this but its nice to be able to call on the struct itself but will move. Unfortunately, tui doesn't support ansi codes directly. So the only way to get the correct formatting is via reimplementing what we do for `fmt`. Plus need some extra stuff to prune unwanted spans.